### PR TITLE
Feat/etherscan

### PR DESCRIPTION
**Pull request by doncesarts**
Issue #https://github.com/mstable/mstable-v1-backlog/issues/43
 Adds ether scan verify feature

### CODE REVIEW

File: tasks/verifyContract.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { task, types } from 'hardhat/config'
+// import { IERC20, IERC20__factory } from 'types/generated'
```
Comment (doncesarts) [inline tasks/verifyContract.ts]: remove code not used

File: utils/etherscan.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  address: string
+  contract?: string
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
+  constructorArguments?: any[]
+  libraries?: {
+    [libraryName: string]: string
+  }
+}
+
+export const verifyEtherscan = async (
```
Comment (doncesarts) [inline utils/etherscan.ts:16]: I would imagine there is a library for this out there in the wild

**Review by doncesarts — COMMENTED** [review]
Well done ser

File: utils/etherscan.ts (hunk 2 above)
Comment (dimsome) [reply utils/etherscan.ts:16]: using the @nomiclabs/hardhat-etherscan, this file is pulled from our current contracts repo

**Pull request merged**